malasakit-django/feature_phone/admin.py

```diff
@@ -16,6 +23,45 @@ class RecordingAdmin(admin.ModelAdmin):
     }
 
 
+    def download_list_action(modeladmin, request, queryset):
+
+
+      def zip_file(zf, obj, file_field, folder=False):
+        if hasattr(obj, file_field) and hasattr(getattr(obj, file_field),
+                                                'url'):
+          fpath = str(os.path.join(os.path.dirname(settings.MEDIA_ROOT),
+                                   str(getattr(obj, file_field).url)[1:]))
+          base = os.path.basename(fpath)
+          if folder:
+            base = file_field + '/' + base 
+          print(base)
```

Comment:
Please refrain from using `print` directly--use the logging framework instead.
```python
LOGGER = logging.getLogger('pcari')

LOGGER.info(...)
```

Also, printing just the basename of the path you are adding isn't very informative, since the logger is shared by all parts of the application. It would be much better to annotate your messages like:
```
Adding files to ZIP archive:
  instructions/001
  instructions/002
```